lightning/src/ln/channelmanager.rs

```diff
@@ -70,6 +70,20 @@ use std::ops::Deref;
 // our payment, which we can use to decode errors or inform the user that the payment was sent.
 
 #[derive(Clone)] // See Channel::revoke_and_ack for why, tl;dr: Rust bug
+#[cfg(test)]
+pub(crate) enum PendingHTLCRouting {
```

Comment:
Would it be reasonable to make `PendingHTLCRouting`'s visibility `pub(super)` given `PendingHTLCInfo` is already so and contains a field of that type?